Package.swift

```diff
@@ -32,8 +32,14 @@ let package = Package(
     targets: [
         .target(name: "RevenueCat",
                 path: "Sources",
-                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"]),
+                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"],
+                linkerSettings: [
+                    .unsafeFlags(["-Xlinker", "-no_application_extension"])
+                ]),
         .target(name: "ReceiptParser",
-                path: "LocalReceiptParsing")
+                path: "LocalReceiptParsing",
+                linkerSettings: [
+                    .unsafeFlags(["-Xlinker", "-no_application_extension"])
```

Comment:
let's add a comment here explaining what this is so maintainers don't have to google it

Reply:
we could link to this https://xcodebuildsettings.com/#application_extension_api_only and explain the need

Reply:
Done